index.ios.js

```diff
@@ -90,7 +91,7 @@ class FBLogin extends Component {
 }
 
 FBLogin.propTypes = {
-  style: View.propTypes.style,
+  style: ViewPropTypes.style,
```

Comment:
May I suggest to use a ternary here to keep the library backwards compatible (< RN 0.49)?

Like this:
```
const viewPropTypes = ViewPropTypes || View.propTypes
FBLogin.propTypes = {
    [...],
    style: viewPropTypes.style,
    [...],
}
```